pypit/data/settings/settings.basespect

```diff
@@ -66,27 +66,16 @@ fits numlamps 1                     # How many lamps are there listed in the hea
 ### Science frames
 science idname OBJECT               # Header key value of science frame for header keyword keyword+idname
 science canbe None                  # If there are frames that will be science ***in addition to other frame types***, include the other frame types here.
-science check condition1 exptime>29 # Arbitrary exptime limit
 
 ### Standard Star frames
 standard idname OBJECT               # Header key value of science frame for header keyword keyword+idname
 standard number 1                    # Number of standard star frames to use
 standard canbe None                  # If there are frames that will be science ***in addition to other frame types***, include the other frame types here.
```

Comment:
The problem is that any check/match condition specified here is applied to all spectrographs, and if you want to overwrite a setting, you need to make sure that the number after `condition` is the same as that specified in the basespect file. Also, there's no way to "delete" a condition once it's specified. So, I think it's safest to specify all of the check/match conditions in the individual settings files of a given spectrograph. It might be a bit cleaner too.